array-manipulation-3/src/intersection.js

```diff
@@ -1 +1,27 @@
 /* exported intersection */
+
+/*
+
+******************************** PSEUDOCODE ********************************
+
+- create an empty array
+- loop through the first array
+- compare each element with each element in the second array
+- if there is a match, add that element to the empty array
+- give back the array
+
+****************************************************************************
+
+*/
+
+function intersection(first, second) {
+  var differenceArray = [];
```

Comment:
`differenceArray` seems like an unusual name given that this is grouping shared data 🤔